backend/routes/auth.js

```diff
@@ -581,6 +608,7 @@ function login(req, res, next) {
 
 			req[C.REQ_REPO].logger.logInfo("User is logged in", responseData);
 
+			// TODO Is this necessary?
```

Comment:
you know it's not :P Check if it's being used, if yes - change it  and remove this, if not - not remove it.